fastlane/Fastfile

```diff
@@ -485,15 +489,26 @@ ENV["validate_translations"]="lintVanillaRelease"
     end
   end
 
-  desc "Import Gutenberg strings"
-  lane :localize_gutenberg do | options |
-    unless options[:skip_module_update] then
-      sh("git submodule init")
-      sh("git submodule update")
+  desc "Import strings from binary dependencies"
+  lane :localize_binary_deps do | options |
+    merge_strings_tool_file_path = File.join(ENV["PROJECT_ROOT_FOLDER"], 'tools/merge_strings_xml.py')
+    binary_imported_libraries.each do  | lib |
+      download_path = android_download_file_by_version(
+        library_name: lib[:name],
+        import_key: lib[:import_key],
+        repository: lib[:repository],
+        file_path: lib[:strings_file_path],
+        github_release_prefix: lib[:github_release_prefix])
+
+      if download_path.nil?
+        UI.error "Abort." unless UI.confirm("Can't download strings file for #{lib[:name]}. Do you want to continue?")
+      else
+        UI.message("Strings.xml file downloaded to #{download_path}.")
+        sh(merge_strings_tool_file_path, lib[:name], download_path)
+        File.delete(download_path) if File.exist?(download_path)
+      end
     end
 
-    sh("cd .. && python ./tools/merge_strings_xml.py")
-
     is_repo_clean = ("git status --porcelain").empty?
     unless is_repo_clean then
       commit_strings(options)
```

Comment:
This logic got me in "trouble".

I run it with an untracked file and, even though I had no string changes, I still got the message to check and commit the strings file.

That might happen in a real execution of the release process lanes, because many have a check to ensure the repo is clean, but I wanted to point it out.

Reply:
Yeah! This is something which may use some improvements. 
Here, I've just kept the same logic we have in the rest of the lanes that work on strings, but we surely want to make it more robust in the future.

Reply:
Sounds good to me. One isolated change at a time 👍

Reply:
I re-read my original comment and noticed I missed a "not"

```diff
- That might happen in a real execution of the release process lanes, because many have a check to ensure the repo is clean, but I wanted to point it out.
+ That might not happen in a real execution of the release process lanes, because many have a check to ensure the repo is clean, but I wanted to point it out.
```